drivers/adc/adc_ene_kb1200.c

```diff
@@ -11,10 +11,13 @@
 #include <zephyr/drivers/pinctrl.h>
 #include <errno.h>
 #include <reg/adc.h>
+#include <zephyr/logging/log.h>
 
 #define ADC_CONTEXT_USES_KERNEL_TIMER
 #include "adc_context.h"
 
+LOG_MODULE_REGISTER(adc, CONFIG_ADC_LOG_LEVEL);
```

Comment:
```suggestion
LOG_MODULE_REGISTER(ene_kb1200, CONFIG_ADC_LOG_LEVEL);
```

Reply:
and to be consistent with all other ADC drivers, it should be `LOG_MODULE_REGISTER(adc_ene_kb1200...`